### PR TITLE
[Merged by Bors] - chore(*): drop some `[Decidable*]` assumptions

### PR DESCRIPTION
**Pull request by urkud**
Related to #10235.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/GroupTheory/Perm/DomMulAct.lean [hunk 1]
```diff
@@ -105,9 +105,11 @@ theorem stabilizer_card [Fintype ι] :
   · exact Finset.prod_congr rfl fun i _ ↦ by rw [Nat.card_eq_fintype_card, Fintype.card_perm]
   · rfl
 
+omit [DecidableEq α] [DecidableEq ι] in
 /-- The cardinality of the set of permutations preserving a function -/
 theorem stabilizer_ncard [Fintype ι] :
     Set.ncard {g : Perm α | f ∘ g = f} = ∏ i, (Set.ncard {a | f a = i})! := by
+  classical
```
Comment (urkud) [inline Mathlib/GroupTheory/Perm/DomMulAct.lean:112]:
```suggestion
```

File: Mathlib/GroupTheory/Perm/DomMulAct.lean [hunk 2]
```diff
@@ -105,9 +105,11 @@ theorem stabilizer_card [Fintype ι] :
   · exact Finset.prod_congr rfl fun i _ ↦ by rw [Nat.card_eq_fintype_card, Fintype.card_perm]
   · rfl
 
+omit [DecidableEq α] [DecidableEq ι] in
```
Comment (urkud) [inline Mathlib/GroupTheory/Perm/DomMulAct.lean:108]:
```suggestion
```

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**